playground/infrastructure/cloudbuild/playground_cd_stable.yaml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+ # contributor license agreements.  See the NOTICE file distributed with
+ # this work for additional information regarding copyright ownership.
+ # The ASF licenses this file to You under the Apache License, Version 2.0
+ # (the "License"); you may not use this file except in compliance with
+ # the License.  You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+steps:
+  - name: ubuntu
+    args:
+      - '-c'
+      - |-
+        echo "Search for CDLOG keyword to find valuable logs entries"
+        echo "CDLOG $(date --utc '+%D %T') Trigger run inputs: 
+        1. PR URL: $_PR_URL; 
+        2. PR Source Repo: $_FORK_REPO; 
+        3. PR Branch: $_PR_BRANCH;
+        4. PR Commit: $_PR_COMMIT; 
+        5. WebHook Action: $_PR_TYPE; 
+        6. Merged to: $_TARGET_PR_REPO_BRANCH; 
+        7. Playground DNS: $_DNS_NAME; "
+        if [[ "${_PR_BRANCH}" != "playground-ci-webhook" ]]; then
+           exit 0
+        fi
+        if [[ ${_TARGET_PR_REPO_BRANCH} == "apache:master" ]]; then
```

Comment:
Reverse the condition. Just exit the script if it's not master bracnh. No need to create if/else trees

Reply:
@rshamunov reversed, as in CI script